### ISSUE TITLE
Cannot download the dataset sucessfully!

### ISSUE THREAD

**Issue by mei123hao** [NVlabs/planercnn#47]
It seems that the datasets cannot be downloaded sucessfully through DropBox .It always failed and lost connection when it is downloading. Can you provides a better way to download the datasets ? Thanks!
![image](https://user-images.githubusercontent.com/36464299/89187080-50df9d00-d5cf-11ea-8b5a-ae10a1dedb9b.png)

**Comment by ajithvcoder** [comment]
@mei123hao  its a 44GB file annotations , did u got the link for images from scannet google groups?

**Comment by RenQJ** [comment]
> @mei123hao its a 44GB file annotations , did u got the link for images from scannet google groups?

excuse me, can you share the data with me? The link seems not to work.

**Comment by NrealWJX** [comment]
> It seems that the datasets cannot be downloaded sucessfully through DropBox .It always failed and lost connection when it is downloading. Can you provides a better way to download the datasets ? Thanks! ![image](https://user-images.githubusercontent.com/36464299/89187080-50df9d00-d5cf-11ea-8b5a-ae10a1dedb9b.png)

I meet the same problem when download by brower and I solve this by ```wget``` with ```-c -t 0```: ```wget -c -t 0 https://www.dropbox.com/s/u2wl4ji700u4shq/ScanNet_planes.zip```

**Comment by Riser6** [comment]
> > It seems that the datasets cannot be downloaded sucessfully through DropBox .It always failed and lost connection when it is downloading. Can you provides a better way to download the datasets ? Thanks! ![image](https://user-images.githubusercontent.com/36464299/89187080-50df9d00-d5cf-11ea-8b5a-ae10a1dedb9b.png)
> 
> I meet the same problem when download by brower and I solve this by `wget` with `-c -t 0`: `wget -c -t 0 https://www.dropbox.com/s/u2wl4ji700u4shq/ScanNet_planes.zip`

Hello，have you finished downloading the complete data?  Every time I get a quarter of the way through the download, it breaks down.

**Comment by NrealWJX** [comment]
> > > It seems that the datasets cannot be downloaded sucessfully through DropBox .It always failed and lost connection when it is downloading. Can you provides a better way to download the datasets ? Thanks! ![image](https://user-images.githubusercontent.com/36464299/89187080-50df9d00-d5cf-11ea-8b5a-ae10a1dedb9b.png)
> > 
> > 
> > I meet the same problem when download by brower and I solve this by `wget` with `-c -t 0`: `wget -c -t 0 https://www.dropbox.com/s/u2wl4ji700u4shq/ScanNet_planes.zip`
> 
> Hello，have you finished downloading the complete data? Every time I get a quarter of the way through the download, it breaks down.

Yes. I meet the exactly same problem with you. I just write a script to automatically re-execute the download command ```wget -c -t 0 https://www.dropbox.com/s/u2wl4ji700u4shq/ScanNet_planes.zip```, and finally I finished downloading.

Remember the parameter ``` -c ``` will continue to download instead of download from scratch.

Best wishes!

**Comment by ajithvcoder** [comment]
Guys you need to mail to `angela.dai@tum.de` . she will send you the link for download it was downloading fine only my computer was not having enough space . you can also download specific files alone there is a script for it
